### PR TITLE
Remove last traces of pygame

### PR DESCRIPTION
**Pull request by nirs**
Some imports and instance variables are not needed now in the client
package, after pygame was removed in #219.

### CODE REVIEW

**Comment by nirs** [comment]
@Ditti4 can you review these changes, completing #219?

File: rose/client/car.py [hunk 1]
```diff
@@ -9,7 +6,6 @@ def __init__(self, id, x, y):
         self.id = id
         self.x = None
         self.y = None
```
Comment (Ditti4) [inline rose/client/car.py]: One off-topic question arises here for me: why not

- do away with the `x` and `y` arguments in the `__init__` or
- assign the values now that we have them.

Obviously this is not related to your change, I only noticed it just now.

Comment (nirs) [reply rose/client/car.py]: @Ditti4 right, the x, y arguments are useless and can be removed - why not send another PR?
